### ISSUE TITLE
Accessibility icon placement

### ISSUE THREAD

**Issue by Doodlepaul** [wmcadigital/wmn-find-metro-zones#24]
There's an issue with the accessibility icon appearing above the text on smaller views. It should be to the left, with the text inset to the right to retain styling from wider views. Text should break rather than icon move above.

![Screenshot 2022-02-11 at 12 10 05](https://user-images.githubusercontent.com/68856858/153589238-efd0849f-cb9c-4819-acd2-d3e152870cc2.png)

Browswer: Safari v15.2
